api-reference/10 UI Components/dxDataGrid/1 Configuration/scrolling/mode.md

```diff
@@ -3,29 +3,38 @@ id: dxDataGrid.Options.scrolling.mode
+A user scrolls a single page only. Scrolling is available only if [all the page's rows](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/#pageSize') do not fit into the UI component's height. In this mode, the [pager](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/pager '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/') performs the main navigation and scrolling is auxiliary. If you disable the [paging](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/'), the UI component loads all rows simultaneously. This action can reduce the UI component's performance. In this case, use another scrolling mode.
 
 - **Virtual**       
-Rows are loaded when they get into the viewport and removed once they leave it. In this mode, users can move to any page instantly.
-
-    If the rows take time to be loaded and rendered, they display gray boxes. [Rendering optimization](/concepts/05%20UI%20Components/DataGrid/10%20Enhance%20Performance%20on%20Large%20Datasets/030%20Rendering%20Optimization.md '/Documentation/Guide/UI_Components/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization') can reduce rendering time and remove the gray boxes. Alternatively, you can disable the **scrolling**.[renderAsync](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/scrolling/renderAsync.md '{basewidgetpath}/Configuration/scrolling/#renderAsync') property, but this can impact performance.
+The component loads pages when they get into the viewport and removes them once they leave it. In this mode, users can move to any page instantly. If the component takes time to load and render rows, they display gray boxes. [Rendering optimization](/concepts/05%20UI%20Components/DataGrid/10%20Enhance%20Performance%20on%20Large%20Datasets/030%20Rendering%20Optimization.md '/Documentation/Guide/UI_Components/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization') can reduce rendering time and remove the gray boxes. Alternatively, you can disable the **scrolling**.[renderAsync](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/scrolling/renderAsync.md '{basewidgetpath}/Configuration/scrolling/#renderAsync') property, but this can impact performance.
 
 - **Infinite**      
-Each next page is loaded once the scrollbar reaches the end of its scale. In this mode, users scroll data gradually from the first to the last page. Row selection in **infinite** and **virtual** scroll mode with Shift+Click does not work in the following cases: 
+The component loads each next once the scrollbar reaches the end of its scale. In this mode, users scroll data gradually from the first to the last page. Row selection in **infinite** and **virtual** scroll mode with Shift+Click does not work in the following cases: 
```

Comment:
`The component loads each next once the scrollbar reaches the end of its scale.`

What do you mean by "The component loads each next"? What does it load next?

Reply:
I simply forgot the word "page", thanks for pointing that out.

Reply:
Thanks, Vlada. I added a suggestion to update your new version (change "each next page" to "the next page").